config/environments/local/local.node.config.toml

```diff
@@ -46,6 +46,7 @@ GlobalQueue = 1024
 		NetProfit = 1
 	    BreakEvenFactor = 1.1			
 		FinalDeviationPct = 10
+		L2GasPriceSuggesterFactor = 0.5
```

Comment:
as a suggestion, since this value is the same as the default, we can avoid adding this line.
it would be good to have in this file only the values different from the default, it would also be good to do a cleanup in the future, but for now we can keep the new values out of the file to make it easier to clean when we are ready to do it.

Reply:
Ok, but until now we have added always to this files all the config parameters. I think we need to continue working in this way (to be consistent) and have a meeting to review if we want to do this in the way you are commenting and if we agree create an issue to cleanup the config files and put only the values that are different from the default (as you propose)